osu.Framework/Testing/TestScene.cs

```diff
@@ -549,6 +549,10 @@ public override void BeforeTest(ITest test)
                 if (test.Fixture is not TestScene testScene)
                     return;
 
+                bool hasSoloAttribute = test.Method?.MethodInfo.CustomAttributes.Any(a => a.AttributeType == typeof(SoloAttribute)) == true;
+                if (DebugUtils.IsNUnitRunning && hasSoloAttribute)
+                    throw new InvalidOperationException($"{typeof(SoloAttribute)} should not be specified on tests running under NUnit.");
```

Comment:
This is gonna be one of the nitpicks of all time, but did you mean

```suggestion
                    throw new InvalidOperationException($"{nameof(SoloAttribute)} should not be specified on tests running under NUnit.");
```

It so happens that `typeof()` also works because of the default implementation of `Type.ToString()`, with the slight distinction that it will print the fully qualified name of the type rather than just the immediate type as `nameof()` does.

Reply:
Yeah i did. Either seems fine your call 😅